app/src/main/java/io/github/droidkaigi/confsched/di/AppModule.java

```diff
@@ -99,6 +100,7 @@ public void change(@NonNull ManualStepMigration.Helper helper) {
                         helper.renameColumn("Session", "categoryId", "category");
                     }
                 })
+                .writeOnMainThread(AccessThreadConstraint.WARNING)
```

Comment:
The current version of Orma raises exceptions if one writes something on main thread.

This setting changes the default behavior from FATAL to WARNING.